src/auth/negotiate/kerberos/Makefile.am

```diff
@@ -10,7 +10,8 @@ include $(top_srcdir)/src/Common.am
 EXTRA_DIST= \
 	README \
 	required.m4 \
-	negotiate_kerberos_auth.8
+	negotiate_kerberos_auth.8 \
+	ext_negotiate_kerberos_acl
```

Comment:
reference to obsolete helper name. needs removing to fix the build errors. This file should now not have any changes.